.github/ISSUE_TEMPLATE/contribute.md

```diff
@@ -1,36 +1,21 @@
 ---
```

Comment:
Adding this file back. I originally removed it because I thought it was hidden for no reason: did not realize it was being used as a github issue template (which users see when they click 'new issue').